web-services-app/src/test/java/edu/unc/lib/boxc/web/services/processing/SingleUseKeyServiceTest.java

```diff
@@ -131,15 +148,11 @@ private void assertDoesNotContainValue(List<CSVRecord> csvRecords, String column
         }
     }
 
-    private void assertCsvContainsCorrectEntry(List<CSVRecord> csvRecords,
-                                               String id, String key, long currentTimestamp) {
+    private void assertCsvContainsCorrectEntry(List<CSVRecord> csvRecords, String id, String key) {
         for (CSVRecord record : csvRecords) {
             if (key.equals(record.get(SingleUseKeyService.ACCESS_KEY))) {
                 assertEquals(id, record.get(SingleUseKeyService.ID));
-                // timestamp must not be null and must be in the future
-                var expirationTimestamp = record.get(SingleUseKeyService.TIMESTAMP);
-                assertNotNull(expirationTimestamp);
-                assertTrue(Long.parseLong(expirationTimestamp) > currentTimestamp);
```

Comment:
Sorry i didn't mean to remove the check about the expiration timestamp being in the future, i just meant that you could set currentTimestamp inside of this method rather than passing it in from outside

Reply:
i thought that was weird but just went with it =D